lib/swaggerValidator/semanticValidator.ts

```diff
@@ -509,13 +510,24 @@ export class SwaggerSemanticValidator {
           visitedParamName.add(name);
 
           if (param.in === "body" || param.in === "formData") {
+            if (param.in === "formData" && param.type === "file") {
+              NumberOfFileTypeFormData++;
+            }
             if (bodyParam !== undefined) {
               const meta = getOavErrorMeta(
                 param.in === bodyParam.in
                   ? "MULTIPLE_BODY_PARAMETERS"
                   : "INVALID_PARAMETER_COMBINATION",
                 {}
               );
+              if (
```

Comment:
How about the case that have multiple formData parameters but without body parameter?

Reply:
parameters sequence isn't fixed, body param might between multiple form data parameters. So don't need to check on the meta.code. And also assign the bodyParam before continue.